RELEASE.md

```diff
@@ -241,13 +241,11 @@ For patch releases issue tracker needs to be created. For patch release, we requ
 * https://github.com/pytorch/pytorch/issues/51886
 
 Only following issues are accepted:
-1. Fixes to regressions against previous major version
+1. Fixes to regressions against previous major version (e.g. regressions introduced in 1.13.0 from 1.12.0 are pickable for 1.13.1)
```

Comment:
Should people interpret "from 1.12.0" == "after 1.12.0"?

Reply:
I think its quite clear 1.12.0 - no regression
1.13.0 - regression is introduced